dotcom-rendering/scripts/webpack/webpack.config.js

```diff
@@ -6,8 +6,10 @@ const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
 const FilterWarningsPlugin = require('webpack-filter-warnings-plugin');
 const LoadablePlugin = require('@loadable/webpack-plugin');
 const { v4: uuidv4 } = require('uuid');
+const WebpackMessages = require('webpack-messages');
 
 const PROD = process.env.NODE_ENV === 'production';
+const DEV = process.env.NODE_ENV === 'development';
```

Comment:
Is there a third state?

```suggestion
const PROD = process.env.NODE_ENV === 'production';
const DEV = !PROD;
```

Reply:
what if there's a `TEST` env? this way you _have_ set the `NODE_ENV` if you want it

Reply:
I think `NODE_ENV` should be `PROD | DEV`. If we're in the testing environment should be in `PROD` to make sure we're testing that environment?

I've found it easier not to overload `NODE_ENV` and rather use `CI` value to check if you're in test. Github actions and other CI [offers this as standard](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables).